pocs/utils/images/__init__.py

```diff
@@ -93,33 +93,30 @@ def _make_pretty_from_fits(fname, **kwargs):
     title = '{} {}'.format(kwargs.get('title', ''), current_time(pretty=True).isot)
 
     new_filename = fname.replace('.fits', '.jpg')
-
     data = getdata(fname)
 
-    norm = ImageNormalize(interval=MinMaxInterval(), stretch=LogStretch())
+    percent_value = 99.81 #(?) | Why this % value? --> Gave a very nice image.
 
+    norm = ImageNormalize(interval=PercentileInterval(percent_value),
+                          stretch=LogStretch())
     wcs = WCS(fname)
+    ax = plt.subplot(projection=wcs)
```

Comment:
This line might need to stay inside the `if`. I'm not sure what happens if you pass a `projection` but don't have a valid WCS. Have you tried on the original images that weren't solved yet?

Reply:
Just for fun I just ran it on an image that a PANOPTES unit just took that has a pretty empty star field and a meteor streak. Looks like it handles the lack of `wcs` fine.

![test](https://user-images.githubusercontent.com/70736/34877525-850f3582-f7fa-11e7-97a8-af2f38bb7763.jpg)

Reply:
There's a lot of dead white space in there around the edges. I think you can do a [`fig.tight_layout()`](https://matplotlib.org/users/tight_layout_guide.html) to get rid of some of that.

Reply:
I tested it on my Jupyter notebook and it seems to work just fine. If I don't assign anything to `projection` then I can't use `coords[]` for the pixel coordinates. I would have to use this for `else`
```ax = plt.subplot()
ax.set_xlabel('X / pixels')
ax.set_xlabel('X / pixels')```
But I think what I have should work just fine :) 
If not then we can change it to this.

Awesome, image looks good, is this with `percent_value = 99.81`? Only asking because maybe we should make that value able to be modified since when I was trying different percentages, and even changing it by 0.1% it changes quite drastically.

Reply:
And I will add the `fig.tight_layout()` soon and let you know how it looks.